test/test-data/sqlalchemy-plugin-query.test

```diff
@@ -0,0 +1,148 @@
+[case queryFromSingleModel]
+# flags: --strict-optional
+
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm.session import Session
+
+Base = declarative_base()
+
+class User(Base):
+    id = Column(Integer, primary_key=True)
+    name = Column(String, nullable=False)
+
+
+session: Session
+
+query = session.query(User)
+reveal_type(query) # N: Revealed type is 'sqlalchemy.orm.query.Query[main.User]'
```

Comment:
I would keep `reveal_type(query)` only in 2-3 test, and switch other to checking some operations on the rows (like you do in some other tests).

Reply:
It's unclear to me how to address this review, and it doesn't seem blocking to me. Can we punt on this for now? It seems more important to me to get this change in.